apps/order/filters.py

```diff
@@ -0,0 +1,16 @@
+import django_filters
+from apps.order.models import BookOrder
+
+
+class BookOrderFilterSet(django_filters.FilterSet):
+
+    class Meta:
+        model = BookOrder
+        fields = ('id',)
```

Comment:
Filter with just `id` might not make that sense since we normally have a query for fetching single objects. Instead, `ids` make sense.

Reply:
@thenav56  These filters are added to fix unusual behaviour with dataloaders, I changed these filter, added title filter in book order and status filter in order.

Reply:
@bimal125 Can you share more about "the unusual behavior with dataloaders"?